proto/cosmos/group/v1/types.proto

```diff
@@ -28,11 +28,27 @@ message Member {
   google.protobuf.Timestamp added_at = 4 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
 }
 
-// Members defines a repeated slice of Member objects.
+
+// MemberRequest represents a group member to be used in Msg server requests.
+// Contrary to `Member`, it doesn't have any `added_at` field
+// since this field cannot be set as part of requests.
+message MemberRequest {
+
+  // address is the member's account address.
+  string address = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+
+  // weight is the member's voting weight that should be greater than 0.
+  string weight = 2;
+
+  // metadata is any arbitrary metadata attached to the member.
+  string metadata = 3;
+}
+
+// Members defines a repeated slice of MemberRequest objects.
 message Members {
 
   // members is the list of members.
-  repeated Member members = 1 [(gogoproto.nullable) = false];
+  repeated MemberRequest members = 1 [(gogoproto.nullable) = false];
```

Comment:
it's a bit weird that we put `MemberRequest` inside `Members`. Maybe the outer one can be called `MembersRequest`?

Or alternatively, can we remove `Members` (or define a non-proto golang struct)? It doesn't seem to be used in any other proto defs.

Reply:
kept it as regular go struct since we define a useful `ValidateBasic` method on it and renamed it to `MemberRequests`